runtime/common/src/main/kotlin/kotlinx/serialization/json/internal/StreamingJsonInput.kt

```diff
@@ -17,11 +17,16 @@ internal class StreamingJsonInput internal constructor(
     public override val context: SerialModule = json.context
     private var currentIndex = -1
     private var entryIndex = 0
-
+    private val configuration = json.configuration
+    
     public override fun decodeJson(): JsonElement = JsonParser(reader).read()
 
     override val updateMode: UpdateMode
-        get() = json.updateMode
+        get() = UpdateMode.OVERWRITE
```

Comment:
Are there any objections to adding `UpdateMode` to `JsonConfiguration`? I know we are going to redesign it, but for now, let's do not touch unrelated things